zerver/webhooks/codeship/doc.md

```diff
@@ -3,19 +3,16 @@ your build statuses.
 
 1. {!create-stream.md!}
 
-2. {!create-bot-construct-url-indented.md!}
+1. {!create-bot-construct-url-indented.md!}
 
-3. Next, for your project, go to **Project Settings**, click on
-   **Notifications**. The URL for the **Notifications** page should
-   look like the following:
+1. Next, on your project's webpage, click on **Project Settings** in
+   the top right corner. Click on the **Notifications** tab.
 
-    `https://codeship.com/projects/PROJECT_ID/configure_notifications`
+1. Click on the **+ New Notification** button.
 
-    where `PROJECT_ID` is the ID of your project in Codeship.
+1. Click on **Webhook**.
```

Comment:
Should this be just part of the previous step, maybe spelled as "Select Webhook"?